File: app/src/main/java/com/akameko/testforupstarts/repository/retrofit/JeansApi.java
package com.akameko.testforupstarts.repository.retrofit;

import androidx.room.Dao;

import com.akameko.testforupstarts.repository.pojos.Jeans;

import java.util.List;

import io.reactivex.Single;
import retrofit2.http.GET;
@Dao
public interface JeansApi {
    @GET("tests/jeans/jeans-default.json")
    Single<List<Jeans>> getJeans();
}

File: app/src/main/java/com/akameko/testforupstarts/repository/retrofit/JeansApi.kt
package com.akameko.testforupstarts.repository.retrofit

import androidx.room.Dao
import com.akameko.testforupstarts.repository.pojos.Jeans
import io.reactivex.Single
import retrofit2.http.GET

@Dao
interface JeansApi {
    @get:GET("tests/jeans/jeans-default.json")
    var jeans: Single<List<Jeans>>
}